brackets-extension/src/command-tau-preview.js

```diff
@@ -126,6 +126,7 @@ define((require, exports, module) => {
             if (doc.isDirty) {
                 if(!window.confirm('Unsaved changed will not be reflected in TAU Design Editor')) {
                     console.log('cancelled');
+					_setSaveMenuEnabled(!isVisible);
```

Comment:
isVisible will be always true because of the condition in line 124
   if (isVisible) {
            console.log('current document', doc);

Reply:
I'd like to avoid magic values in the code produced by me. This `!isVisible` is state-changing returned by this function.